04week/nasa/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel="stylesheet" href="./css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Noto+Sans" rel="stylesheet">
+</head>
+
+<body>
+  <span class="nasa"> <h1 class="text">NASA</h1>
+</span>
+  <!-- main section with 3 divs -->
+  <main>
+    <div class="social">
+      <h2>Social Media</h2>
+      <p>
+        Twitter- @nasa <br /> Facebook- fb.com/nasa <br /> Instagram- @nasa <br /> Youtube- youtube.com/NASA
```

Comment:
I would have made each line a separate element, that way you can have control over the spacing, layout and padding.